### ISSUE TITLE
Pokemon Stats dropping on Level-Up

### ISSUE THREAD

**Issue by kevsgrove** [Ajarmar/universal-pokemon-randomizer-zx#26]
Playing through a randomized version of FireRed, I noticed that Snorlax's max HP was strangely low.

I saved and used a savegame editor to debug, but when opening Snorlax in the editor, his stats are set to what they should be.

However, after playing a while, I noticed his max HP was very low again, and found that the stats were being skewed upon each level-up. It's as if the game is calculating his stats based on some other pokemon's base stats.

Picture of level-up screen:
[https://i.imgur.com/tVCvsE5.png]()

TLDR: Stats of one specific pokemon get scrambled upon level-up.

**Comment by Ajarmar** [comment]
Did you choose to create a log file when performing the randomization? If so, can you upload it here?

**Comment by kevsgrove** [comment]
I did not create a logfile, although I think I realized it was my own mistake. I believe I accidentally replaced the .rom with one that had the base stat randomization checked. Re-randomizing the base .rom fixed my issue.

**Comment by Ajarmar** [comment]
Glad you solved it!
